### PR TITLE
chore: gitignore and prettierignore

### LINKED ISSUE #52: Prettier readme
Are you fine with making the readme markdown Prettier compliant?

It mostly wants to remove extra whitespace, use "`" over "~" and prefer "-" over "*". I can make a separate PR for that. It would make contributing changes easier.

### PR DESCRIPTION
**Pull request by brandonkal**
### Which issue does this fix?

Closes #52

### Describe the solution

Ignores .vscode
Ignores other node things.
Configures Prettier to ignore markdown files.

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.md
```
Comment (jakedeichert) [inline .prettierignore]: Adding a prettier config to this Rust repo is really odd. By this logic, this repo could be filled with config files that are unrelated to this ecosystem 😅 

Lets remove this prettierignore file and add it to gitignore so you can personally use this file without it being committed. Exact same logic as ignoring vscode editor configs 👍

Comment (brandonkal) [reply .prettierignore]: Really?
It's a single line hidden file that declares your stated preference regarding the common tool and will save you from reviewing diffs that revert your unique formatting preferences but okay...

Comment (jakedeichert) [reply .prettierignore:1]: Yes please
